src/models/constants.js

```diff
@@ -22,6 +22,7 @@
  ******/
 'use strict';
 
+const partyAccountsTable = 'account';
```

Comment:
this party account logic is from phase 1. we can probably remove it here and throughout.

Reply:
Cool, yeah I thought there would be a bit of extra stuff to clean up.

I'll tackle that in a followup PR to make this one smaller.